src/charts/chartWindow.js

```diff
@@ -42,7 +42,6 @@ define(["jquery","windows/windows", "text!charts/chartWindow.html", 'lodash', "j
                 },
                 resize: _trigger_Resize_Effects
             }, options );
-
```

Comment:
@4p00rv Looks like this file change is not needed. Please revert. 
Its better to submit files which really has some change so that I get an idea of how many files you touched for particular task. This gives me some idea of impact of such PR